src/main/java/org/folio/rest/impl/InitAPIs.java

```diff
@@ -41,19 +46,36 @@ public void init(Vertx vertx, Context context, Handler<AsyncResult<Boolean>> res
         }
       });
 
-      // Initialize data for helpers
-      AbstractHelper.init();
-
       // Initialize ResponseWriter and check if jaxb marshaller is ready to operate
       if (!ResponseHelper.getInstance().isJaxbInitialized()) {
         throw new IllegalStateException("The jaxb marshaller failed initialization.");
       }
 
-      resultHandler.handle(Future.succeededFuture(true));
+      // Initialize data for OAI-PMH endpoint helpers
+      context.config().put(IDENTIFIER_PREFIX, createRepositoryIdentifierPrefix());
```

Comment:
Instead of the static variable, I decided to introduce config usage via vert.x context. This will allow in future to implement approach to update configs [periodically](https://github.com/folio-org/raml-module-builder#adding-code-to-run-periodically).

Reply:
That's good alternative approach. The only disadvantage is that you need access to the context to get property value.
Let's stick with it for now. If we see it causes inconveniences, will think about other ways then.